src/components/BookCard.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+function BookCard() {
+  return (
+    <div>
+      <li>
+
+        Mirwais
+        :
+        Java Programming
+      </li>
+      <button type="button"> Remove</button>
+    </div>
+  );
+}
```

Comment:
- [ ] Here you have to take book data as `prop` and render the title and the author.